app/Http/Controllers/Auth/LoginController.php

```diff
@@ -37,4 +37,10 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    // ログイン後のリダイレクトをマイページに
+    public function redirectPath()
+    {
+        return '/mypages';
```

Comment:
ここ飛びますかね？:eyes:

```suggestion
       return redirect('/mypages');
```

かな、と思いましたが無事飛ぶようでしたら問題ないです！